util/settings/settings.go

```diff
@@ -1735,6 +1741,31 @@ func (a *ArgoCDSettings) IssuerURL() string {
 	return ""
 }
 
+func (a *ArgoCDSettings) EnableUserInfoGroups() bool {
```

Comment:
Nit: I think `UserInfoGroupsEnabled` or `IsEnableUserInfoGroups` would make more sense as the method name here, because `EnableUserInfoGroups` could suggest an imperative to enable the settings.

Reply:
Oh yeah that makes much more sense, changed it to `UserInfoGroupsEnabled`